packages/dev-server-esbuild/package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@mdn/browser-compat-data": "^4.0.0",
     "@web/dev-server-core": "^0.3.19",
-    "esbuild": "^0.12 || ^0.13 || ^0.14",
+    "esbuild": "^0.16 || ^0.17",
```

Comment:
If there are no consumed APIs changing as part of this update, is there anything wrong with leaving the previous verions? `0.14.0` is at least a year old, so I'm not against dropping them, just wanted to get your thoughts. As a "breaking" change (patches before 1.0) it's not an issue either way.

Reply:
I suppose we could just leave the old ones. On the other hand; *some* old ones are broken, so we could prevent anyone from going down the same rabbit hole?

Reply:
Oh, yeah, if they are broken, let's use the breaking semantics of the change you've listed and exclude them. Great call!